commpy/channelcoding/README.md

```diff
@@ -106,9 +107,9 @@ noiseVar = 10**(-snrdB/10) # noise variance (power)
 
 N_c = 10 # number of trials
 
-BER_soft = np.empty((N_c,))
-BER_hard = np.empty((N_c,))
-BER_uncoded = np.empty((N_c,))
+BER_soft = -9e99*np.ones(N_c)
+BER_hard = -9e99*np.ones(N_c)
+BER_uncoded = -9e99*np.ones(N_c)
```

Comment:
This looks good to me. Could you just explain why you removed `empty` that should work here?

Reply:
`empty` works indeed, but it produces arrays filed with unknown values, I don't find it very elegant. I always prefer working with arrays filled with know values. But if you do not like it, I can remove it.

Reply:
I get your point. I not keen on this initialisation as it suggest that -9e99
is intended. I you want to know the initialisation pattern, I prefer to fill
with `np.nan` or tick to `np.empty`. That way, it is straightforward that the
initialisation does not matter on the following code.

Reply:
Should be fixed now